app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :comments, foreign_key: 'author_id', class_name: 'Comment'
+  has_many :posts, foreign_key: 'author_id', class_name: 'Post'
+  has_many :likes, foreign_key: 'author_id', class_name: 'Like'
+
+  def last_three_posts
+    posts.last(3)
+  end
```

Comment:
- [ ] To return the _most recent 3_ posts in order, our fetch request implementation should be similar to:
```ruby
def recent_post
  posts.order('created_at Desc').limit(3)
end
```
Kindly ensure to improve upon the same.

Reply:
Hello @rloterh, can you tell me why I need to change this? It works properly